test/torch/federated/test_dataset.py

```diff
@@ -25,6 +29,24 @@ def test_base_dataset(workers):
     with pytest.raises(AttributeError):
         assert dataset.targets.location.id == 0
 
+    transform_dataset = BaseDataset(inputs, targets)
+
+    def func(x):
+
+        return x * 2
+
+    transform_dataset.transform(func)
+
+    expected_val = th.tensor([2, 4, 6, 8])
+
+    ten = []
+
+    for i in transform_dataset:
```

Comment:
You can replace this for loop with one statement:

transformed_val = [val[0].item() for val in transform_dataset]